### PR TITLE
Adding more tests for DataLake

### PR DESCRIPTION
**Pull request by begoldsm**
Adding ACL tests for filesystem
Adding Catalog tests
Renaming files to ensure everything is DataLake*

### CODE REVIEW

**Comment by azurecla** [comment]
Hi __@begoldsm__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (begoldsm). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

File: src/ResourceManagement/DataLake.AnalyticsCatalog/DataLakeAnalyticsCatalog.Tests/Properties/AssemblyInfo.cs [hunk 1]
```diff
@@ -1,15 +1,14 @@
 ﻿using System.Reflection;
```
Comment (stankovski) [inline src/ResourceManagement/DataLake.AnalyticsCatalog/DataLakeAnalyticsCatalog.Tests/Properties/AssemblyInfo.cs]: Copyright header

**Comment by stankovski** [comment]
@yugangw-msft please merge when you are ready

**Comment by stankovski** [comment]
@begoldsm please pull from upstream and update the PR

**Comment by begoldsm** [comment]
I am closing this PR as we are re-opening it in the master branch.

**Pull request closed without merging**